packages/core/src/components/map-controls/map-controls.tsx

```diff
@@ -105,9 +105,9 @@ export class MapControls {
               <dso-icon icon="times"></dso-icon>
             </button>
           </header>
-          <div class="content">
+          <dso-scroll-container>
```

Comment:
Ik twijfel of de `<dso-scroll-container>` zelf padding moet hebben of dat hier de `.content` class terug moet zodat het component daar controle over heeft.

Reply:
Dat moet bij de inhoud liggen